src/modules/auth/clients/ClientDetailContainer.js

```diff
@@ -108,6 +108,8 @@ const ClientDetailContainer = (props) =>  {
 			body.client_name = "";
 		}
 
+		setDisabled(true);
```

Comment:
@Aringoaway Put `setDisabled(true)` between lines 92 and 93 (before `body` declaration)